app/pages/cif/project-revision/[projectRevision]/form/[formIndex]/index.tsx

```diff
@@ -3,17 +3,9 @@ import { Button } from "@button-inc/bcgov-theme";
 import DefaultLayout from "components/Layout/DefaultLayout";
 import TaskList from "components/TaskList";
 import { TaskListMode } from "components/TaskList/types";
-import { useFormPages } from "data/formPages/formStructure";
-import useRedirectTo404IfFalsy from "hooks/useRedirectTo404IfFalsy";
-import useRedirectToLatestRevision from "hooks/useRedirectToLatestRevision";
-import useRedirectToValidFormIndex from "hooks/useRedirectToValidFormIndex";
 import withRelayOptions from "lib/relay/withRelayOptions";
-import { useCreateProjectRevision } from "mutations/ProjectRevision/createProjectRevision";
 import { useRouter } from "next/router";
-import {
-  getProjectRevisionFormPageRoute,
-  getProjectRevisionPageRoute,
-} from "routes/pageRoutes";
+import { useFormIndexHelpers } from "../../../../../../hooks/useFormIndexHelpers";
```

Comment:
We shouldn't need all the ../../.. here
```suggestion
import { useFormIndexHelpers } from "hooks/useFormIndexHelpers";
```